### PR TITLE
Fixed Buttons style issues

### PR DESCRIPTION
**Pull request by kinloveko**
• Questions - Ken - The "Add category" button needs to have a class. - DONE ✅ 
• Questions - Ken - The pagination text for the number of questions is incorrect. -  DONE ✅
• Questions - Ken - Set pagination text number. -  DONE ✅
• Home Page - Ken - add classname to log-in button, review   -DONE ✅
• QuizRoom - Ken/Xerxes - Add Pagination or Remove Pagination, Frontend only - DONE ✅

### CODE REVIEW

File: WebApp/frontend/quiz-master/app/dashboard/card.tsx [hunk 1]
```diff
@@ -95,27 +79,20 @@ export default function Card() {
...
-        };
-    
-        fetchQuestionsLength();
-      }, []); // Empty dependency array to run only on mount
-    
+
+      const fetchData = async () => {
+        try {
+          const response = await GetAllQuestion(); // Wait for the promise to resolve
+          console.log("RESPONSE1121:", response );
```
Comment (jaymar921) [inline WebApp/frontend/quiz-master/app/dashboard/card.tsx:86]: what the f is this HAHAA

File: WebApp/frontend/quiz-master/app/quiz-rooms/create-quiz-room/page.tsx
Comment (jaymar921) [inline WebApp/frontend/quiz-master/app/quiz-rooms/create-quiz-room/page.tsx:1]: nice

**Review by jaymar921 — APPROVED** [review]
looks good to me

**Review by rbacordio — APPROVED** [review]

File: WebApp/frontend/quiz-master/app/dashboard/card.tsx (hunk 1 above)
Comment (kinloveko) [reply WebApp/frontend/quiz-master/app/dashboard/card.tsx:86]: we call it ART! HAHAHAHAHA

**Pull request merged**